packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -464,7 +464,7 @@ function NoteEditor(props: NoteEditorProps) {
 		noteToolbarButtonInfos: props.toolbarButtonInfos,
 		plugins: props.plugins,
 		fontSize: Setting.value('style.editor.fontSize'),
-		fontFamily: Setting.value('style.editor.fontFamily'),
+		fontFamily: `"${Setting.value('style.editor.fontFamily')}", ` + 'Avenir, Arial, sans-serif',
```

Comment:
```suggestion
		fontFamily: `"${Setting.value('style.editor.fontFamily')}", Avenir, Arial, sans-serif`,
```

String concatenation shouldn't be necessary here.

Additionally, it may make sense to use `JSON.stringify` here because it escapes internal `"`s:
```suggestion
		fontFamily: `${JSON.stringify(Setting.value('style.editor.fontFamily'))}, Avenir, Arial, sans-serif',
```

Reply:
Oh yes, you are right. thank you!